webpack/assets/javascripts/react_app/routes/Models/ModelsPage/ModelsPage.js

```diff
@@ -1,16 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Button } from '@patternfly/react-core';
-import { Link } from 'react-router-dom';
 
 import { translate as __ } from '../../../common/I18n';
-import PageLayout from '../../common/PageLayout/PageLayout';
+import PageLayout from '../../../components/PF4/TableIndexPage/TableIndexPage';
```

Comment:
I don't think we need to use a separate PF4 dir.
eventually, all components will be PF4, and I would vote for removing the old `PageLayout`

Reply:
If were keeping the `TableIndexPage` name I can move it out :)